benchmarking/cli/benchmark_factory.py

```diff
@@ -35,7 +35,7 @@
     resnet_cifar10_benchmark,
     resnet_cifar10_default_params,
 )
-from benchmarking.nursery.lstm_wikitext2.definition_lstm_wikitext2 import (
+from benchmarking.definitions.definition_lstm_wikitext2 import (
```

Comment:
Do we really want this in the benchmarks? Are we planning to use this in the future, or rather switch to a transformer based benchmark?

Reply:
Do you think this benchmark is not good?

I'd like to have it, at least until there is something else more or less equivalent in terms of running time and difficulty.